pkg/virt-api/rest/subresource.go

```diff
@@ -1622,3 +1623,29 @@ func (app *SubresourceAPIApp) SEVFetchCertChainRequestHandler(request *restful.R
 
 	app.httpGetRequestHandler(request, response, validate, getURL, v1.SEVPlatformInfo{})
 }
+
+func (app *SubresourceAPIApp) SEVQueryLaunchMeasurementHandler(request *restful.Request, response *restful.Response) {
+	if !app.ensureSEVEnabled(response) {
+		return
+	}
+
+	validate := func(vmi *v1.VirtualMachineInstance) *errors.StatusError {
+		if !vmi.IsRunning() {
```

Comment:
In other similar functions you have `!vmi.IsScheduled()` condition. Why is it different here?